worker/worker.go

```diff
@@ -28,15 +27,15 @@ type Worker interface {
 	// ResolveOp resolves Vertex.Sys() to Op implementation.
 	ResolveOp(v solver.Vertex, s frontend.FrontendLLBBridge, sm *session.Manager) (solver.Op, error)
 	ResolveImageConfig(ctx context.Context, ref string, opt llb.ResolveImageConfigOpt, sm *session.Manager, g session.Group) (digest.Digest, []byte, error)
-	// Exec is similar to executor.Exec but without []mount.Mount
-	Exec(ctx context.Context, meta executor.Meta, rootFS cache.ImmutableRef, stdin io.ReadCloser, stdout, stderr io.WriteCloser) error
 	DiskUsage(ctx context.Context, opt client.DiskUsageInfo) ([]*client.UsageInfo, error)
 	Exporter(name string, sm *session.Manager) (exporter.Exporter, error)
 	Prune(ctx context.Context, ch chan client.UsageInfo, opt ...client.PruneInfo) error
 	GetRemote(ctx context.Context, ref cache.ImmutableRef, createIfNeeded bool) (*solver.Remote, error)
 	FromRemote(ctx context.Context, remote *solver.Remote) (cache.ImmutableRef, error)
 	PruneCacheMounts(ctx context.Context, ids []string) error
 	ContentStore() content.Store
+	GetExecutor() executor.Executor
+	GetCacheManager() cache.Manager
```

Comment:
Using the `Get` prefix here to avoid naming conflict with `base.WorkerOpt.{Executor,CacheManager}` embedded in `base.Worker`.  We can change the names in`WorkerOpt` if the `Get` prefix is undesirable.

Reply:
Why is there a collision if `Executor` is part of `WorkerOpt`.  Isn't it just

```
func (w *Worker) Executor() executor.Executor {
	return w.WorkerOpt.Executor
}
```

Reply:
Yeah, that works, we will have to also change the ~dozen references to w.Executor like:
```
return ops.NewExecOp(v, op, baseOp.Platform, w.CacheManager, sm, w.MetadataStore, w.Executor, w)
```

Reply:
We can do this as follow-up if you like but generally we should avoid `Get*`

Reply:
np, was not sure what your preference for this was, will fix